allennlp/service/servable/models/semantic_role_labeler.py

```diff
@@ -34,16 +36,24 @@ def __init__(self):
                 })
 
         self.model = SemanticRoleLabeler.from_params(self.vocab, params)
+        self.nlp = spacy.load('en', parser=False, vectors=False, entity=False)
 
     def predict_json(self, inputs: JSONDict) -> JSONDict:
-        sentence = self.tokenizer.tokenize(inputs["sentence"])
-        text = TextField(sentence, token_indexers=self.token_indexers)
-        # TODO(joelgrus) use spacy to identify verbs
-        results = {"idx": []}  # type: Dict[str, Any]
-        for i in range(len(sentence)):
-            verb_indicator = IndexField(i, text)
-            output = self.model.tag(text, verb_indicator)
-            output["class_probabilities"] = output["class_probabilities"].tolist()
-            results["idx"].append(output)
+        sentence = inputs["sentence"]
+        tokens = self.tokenizer.tokenize(sentence)
+        text = TextField(tokens, token_indexers=self.token_indexers)
+
+        results = {"verbs": []}  # type: JSONDict
```

Comment:
I hadn't noticed this `JSONDict` type before; I think it's a good idea, and should probably just go in `allennlp.common`, instead of `allennlp.service.servable`.  Though I would vote for [`JsonDict`](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case) over `JSONDict`...  And I would definitely recommend doing that in a separate PR, whenever someone gets to it.